docs/snippets/azkv-datafrom-external-secret.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: external-secrets.io/v1alpha1
+kind: ExternalSecret
+metadata:
+  name: example
+spec:
+  refreshInterval: 1h           # rate SecretManager pulls Azure Key Vault
+  secretStoreRef:
+    kind: SecretStore
+    name: example               # name of the SecretStore (or kind specified)
+  target:
+    name: secret-to-be-created  # name of the k8s Secret to be created
+    creationPolicy: Owner
+  dataFrom:
+  - find:
+      name:
+        regexp: "^example"
+  - find:
+      tags:
+        author: seb
+        environment: dev
+  - extract: 
+      key: test
+      metadataPolicy: Fetch # get all tags of the secret test
```

Comment:
For this manifest to work, it needs to be `apiVersion: external-secrets.io/v1beta1`.